src/lexer.c

```diff
@@ -71,7 +67,16 @@ typedef enum {
     T_case,
     T_break,
     T_default,
-    T_continue
+    T_continue,
+    T_preproc_include,
```

Comment:
Use `cppd` to stand for C PreProcessor Directives. That is, `T_cppd_include`, `T_cppd_define`, etc.